anaconda_build/README.md

```diff
@@ -74,57 +74,49 @@ local development. E.g. on uframe-3-test, mi-instrument is installed under the r
 $ conda install mi-instrument
 ```
 
-## The stream/ingest engines and environment
+## The stream/ingest recipes, packages and environment
 
-The recipes are located in the <i>enginemeta</i>, <i>ingestEngine</i> and <i>streamEngine</i> folders. The enginemeta recipe declares the dependencies used while installing the packages built from the other recipes (ingestEngine and streamEngine). Modify it only as those dependencies change. The other recipes are based on the whole codebases via clones of the following Git repositories.
+These recipes are located in the <i>enginemeta</i>, <i>ingest_engine</i> and <i>stream_engine</i> folders. The enginemeta recipe declares the dependencies required for use in an environment that can be used to run the code obtained from the ingest_engine and stream_engine recipes. Modify it only as those dependencies change.
 
-* git clone ssh://uframe-cm.ooi.rutgers.edu:29418/ingest_engine
-* git clone https://github.com/oceanobservatories/stream_engine
+The ingest_engine and stream_engine recipes declare the enginemeta package as a dependency. The packages built from them are named the same as the recipes. They obtain and install their respective Git repositories from a tagged checkout point (that's specified in the recipe) into their respective packages:
 
-The ingestEngine and streamEngine recipes declare the enginemeta package as a dependency. When built, these recipes expect the code from the corresponding repositories to be pulled into the builder's $HOME/uframes/engines folder and the appropriate checkout locations activated as
+* ssh://uframe-cm.ooi.rutgers.edu:29418/ingest_engine
+* https://github.com/oceanobservatories/stream_engine
 
-```bash
-$ cd $HOME/uframes/engines/stream_engine
-$ git fetch
-$ git checkout v1.3.7
-$ git submodule update --init --recursive
-$ cd $HOME/uframes/engines/ingest_engine
-$ git fetch
-$ git checkout build_8.0_20170630
-```
-
-Once done they need to be TAR/bzip'd there. Whenever the code bases represented by these repositories are modified, these TARs need to be updated. Use the following commands to TAR/bzip them
+These recipes must be obtained from the ooi-config repository, modified as needed and built locally from the anaconda_build folder as 
```

Comment:
you could make this a markdown url pointing to the github location of the repository

Reply:
I modified the verbiage from "the ooi-config repository" to "this repository" and made it into a markdown url.